### DISCUSSION TITLE
Ways to speed up training

### DISCUSSION THREAD

**Discussion by oTree-org** [TencentARC/GFPGAN#272] [General]
Hi all,

I'm training with a dataset of 12,000 images, following the instructions in the "Training" section of the README

- ETA is currently 7 days.
- GPU is a single RTX 3090 (stays at around 10%) 
- CPU is an E5-2630 @ 2.20GHz (usually around 10% but spikes to 40% a few times a minute)
- RAM & GPU RAM are at a small fraction of system limit
- Using train_gfpgan_v1_simple.yml with no colorization and no validation.

Is there a way to speed up the training, by increasing GPU/CPU usage, or finetuning an existing model, or something else?

I'd also be interested to hear how long it took for others to train with their datasets.

**Comment by AIhasArrived** [comment]
hey @oTree-org , please answer me and do not ignore me. Were you able to make it use GPU? I am also seeing only 10% usage and I find it a bit depressing to have it that slow.
So did you find a way to make it use GPU and acceleate it please?
Please answer.
